### ISSUE TITLE
What does nerf training speed have to do with?

### ISSUE THREAD

**Issue by qhdqhd** [NVlabs/instant-ngp#663]
What does nerf training speed have to do with? My experiment found that it has a lot to do with the resolution of the image, and it may also be related to the number of images and the complexity of the scene?

**Comment by Hezhexi2002** [comment]
Yes,it dose,because the resolution of your images will determine wether your cuda memory is enough or not,same to the number of images and the complexity of the scene
